internal/report.go

```diff
@@ -434,6 +458,38 @@ func IgnoredStatements(conv *Conv) (l []string) {
 	return l
 }
 
+func reportNameChanges(conv *Conv, w *bufio.Writer) {
+
+	w.WriteString("-----------------------------------------------------------------------------------------------------\n")
+	w.WriteString("Name Changes in Migration\n")
+	w.WriteString("-----------------------------------------------------------------------------------------------------\n")
+	fmt.Fprintf(w, "%25s %15s %25s %25s\n", "Source Table", "Change", "Old Name", "New Name")
```

Comment:
nit/Suggestion:you can also do 
`w.Writestring(fmt.Sprintf("%25s %15s %25s %25s\n", "Source Table", "Change", "Old Name", "New Name"))`
as it is consistent with the rest.

Reply:
Are there any benefits of using w.WriteString?

Reply:
will do this.